### ISSUE TITLE
Call to undefined method getDriver

### ISSUE THREAD

**Issue by pbatroff** [systopia/CiviProxy#75]
In some configurations, specifically when sending a test Mail from outgoing Emails (civicrm/admin/setting/smtp?reset=1), the following Error Message happens: 

```
Error: Call to undefined method CRM_Civiproxy_Mailer::getDriver() in CRM_Utils_Mail::sendTest()
```

I am not sure when this also is triggered, but to prevent other future errors the driver should be passed to the Mailer class, and the function should be implemented in CRM_Civiproxy_Mailer.

**Comment by pbatroff** [comment]
@bjendres FYI

**Comment by pbatroff** [comment]
I merged the fix to Main and released 1.0.0-alpha1. SendMail test worked well, no white screen anymore.
